docs/users/intro.md

```diff
@@ -0,0 +1,94 @@
+
+The result is a social media site that's part-forum, part-Discord, part-LJ clone, and yet none of those at all.
+
+## How BobaBoard is Different...
+We hear you: you already have 37 social accounts and maybe use 3 of them, if you squint, why would you want another? Why give another startup platform a chance? What's _different_ about this place?
+
+### Realms
+:::info
+
+This section of the guide is still in development! We're still in the process of figuring out how Realms will work. The general idea is explained below, but your input will help shape things!
+
+:::
+
+BobaBoard is made up of distinct areas called **Realms**. Like LJ and its clones (and to some extent, Discord), each Realm is its own domain within the BobaBoard "network". Each Realm may have different goals, cultures, and rules.
+
+As BobaBoard expands to be open to the public, Moderators will be able to establish their own Realms. Users will have to join each Realm they want posting and reading access to and explicitly agree to their rules.
+
+There are currently two Realms making up BobaBoard:
+- V0: The Alpha
+  > The OG BobaBoard. A closed Alpha for Ms. Boba's friends, Boba volunteers, and early adopters.
+
+- Fandom Coders
+  > The first official "Realm". It's a community of fandom people who also code or want to learn to code, particularly if they use coding in their fandom activities!
+
+For more information on each Realm, visit their sections in the guide.
+
+### Boards, Not Blogs
+Each **Realm** is made up of multiple Boards. Boards are similar to the boards in a forum or modern Discord channels; they define the broad topics of conversations and give users a heads-up as to what they can expect to find.
+
+To interact in the Realm, you simply post to whatever you find most appropriate. Don't worry about getting it wrong: there is no wrong. If something might fit in more than one place, simply choose the one that's most appropriate according to your own nebulous criteria.
+
+_"The boards are more what you'd call guidelines than actual rules."_
+
+Unlike other fandom spaces, there's no personal "blog" or "page" on BobaBoard. There's no user address that you can give someone so that they can see all your posts. You won't have followers, friends, or mutuals.
+
+### Random Anonymous Identities
+Identities are assigned at random each time a user starts a post, comments, or contributes. Initially, you might be confused by seeing the same anonymous identities all around the space. Just know that, outside a single thread, *the users behind these identities are not the same*. There's not a single "Revolver Ocelot (Revolver Ocelot)": every new thread each identity is shuffled and reassigned to someone else. 
```

Comment:
I think this explanation is very similar to the previous guide's, which many found confusing. Maybe let's workshop a new one together?

Reply:
Yeah, I struggled a lot with this and just tried to edit the original guide. It's very difficult to explain. I wonder if screenshots might help?

Reply:
Can you start a convo about this in the volunteers server? or even v0

Reply:
I've commented this section out for now, until we can get some brainstorming happening.